File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/lighthouses/domain/LighthousesLighthouseObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.lighthouses.domain;

import com.zwstudio.logicpuzzlesandroid.common.domain.AllowedObjectState;

public class LighthousesLighthouseObject extends LighthousesObject {
    public AllowedObjectState state = AllowedObjectState.Normal;
    public String objAsString() {
        return "lighthouse";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/lighthouses/domain/LighthousesLighthouseObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.lighthouses.domain

import com.zwstudio.logicpuzzlesandroid.common.domain.AllowedObjectState

class LighthousesLighthouseObject : LighthousesObject() {
    var state = AllowedObjectState.Normal
    override fun objAsString(): String {
        return "lighthouse"
    }
}